### ISSUE TITLE
contrib/labstack/echo: support v4

### ISSUE THREAD

**Issue by kuchlein** [DataDog/dd-trace-go#543]
Echo recently changed the way that they package and ship their code. Specifically version 4 uses a new import scheme (see [echo git](https://github.com/labstack/echo) where they'd like you to use an import of `github.com/labstack/echo/v4` rather than `github.com/labstack/echo` which was used before. In order to get the contributed code to work I had to make a minor change to the imports in echotrace.go from:

```
import (
	"math"
	"strconv"

	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"

	"github.com/labstack/echo"
)
```
to
```
import (
	"math"
	"strconv"

	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"

	echo "github.com/labstack/echo/v4"
)
```
I'm not sure how you want to handle this but it would be helpful to add support for Echo V4?

**Comment by gbbr** [comment]
Sure. Are you willing to commit to a PR? We should duplicate the folder and create a `contrib/labstack/echo.v4`  as per the guidelines I have recently added [here](https://github.com/DataDog/dd-trace-go/tree/v1/contrib#usage).

**Comment by gbbr** [comment]
#672 has been merged, so we should be able to do this work now, if there is still interest.

**Comment by gbbr** [comment]
There's a [v1.25.0-rc2](https://github.com/DataDog/dd-trace-go/releases/tag/v1.25.0-rc2) out for testing. It contains [a few other changes too](https://github.com/DataDog/dd-trace-go/compare/v1.24.1...v1.25.0-rc2). Feel free to try it out and give feedback. This does not mean that we freeze master and have started to prepare for releasing 1.25. It is just an intermediate testing step which requires tagging. Development and merging PRs should continue as usual.

This tag also contains an integration for an integration for `github.com/labstach/echo/v4` [here](https://github.com/DataDog/dd-trace-go/tree/v1.25.0-rc2/contrib/labstack/echo.v4) which only works with modules enabled.

**Comment by knusbaum** [comment]
This has been merged in #698 and will be released in `v1.26.0`
